lib/node.js

```diff
@@ -47,6 +155,40 @@ class Node {
     }
     this._namespace = namespace;
```

Comment:
ditto. Remove this._namespace?

Reply:
My vote is yes. @minggangw may have some historical perspective that we need to consider.

Reply:
We can remove the |_namespace| and |_name| now as we have the binding methods which can get them through rcl. So please remove them. Thanks for implementing the series of param related features, I may spend some time to have a look and will give comments when it's finished, thanks!

Reply:
done.